testing/test_package/pubspec.yaml

```diff
@@ -4,5 +4,6 @@ description: Best package ever.
 version: 0.0.1
 dependencies:
   meta: ^1.0.0
+  tuple: 1.0.1
```

Comment:
I'm guessing you're intending to pin tuple and don't expect that to introduce constraint solving problems down the road?